test/runtests.jl

```diff
@@ -26,71 +36,68 @@ function orient(a, b, c)
+    @test orient(a, b, c) == CPredicates.orient(a, b, c)
+    @test_broken orient(a, b, c) == ExactPredicates.orient(a, b, c)
 
-    (a,b,c) = (a = 0.0 + 0.0im, b = 0.0 + 5.0e-324im, c = 5.060829986287029e79 + 3.552170572284382e228im)
-    @test orient(a,b,c) == CPredicates.orient(a, b, c)
-    @test_broken orient(a,b,c) == ExactPredicates.orient(a,b,c)
+    (a, b, c) = (a=0.0 + 0.0im, b=0.0 + 5.0e-324im, c=5.060829986287029e79 + 3.552170572284382e228im)
+    @test orient(a, b, c) == CPredicates.orient(a, b, c)
+    @test_broken orient(a, b, c) == ExactPredicates.orient(a, b, c)
 
-    (a, b, c) = (a = 0.0 + 0.0im, b = 0.0 + 5.0e-324im, c = 5.0e-324 + 0.0im)
-    @test orient(a,b,c) == CPredicates.orient(a, b, c)
-    @test_broken orient(a,b,c) == ExactPredicates.orient(a,b,c)
+    (a, b, c) = (a=0.0 + 0.0im, b=0.0 + 5.0e-324im, c=5.0e-324 + 0.0im)
+    @test orient(a, b, c) == CPredicates.orient(a, b, c)
+    @test_broken orient(a, b, c) == ExactPredicates.orient(a, b, c)
 
-    (a,b,c) = (a = 0.0 + 0.0im, b = 0.0 + 5.495397811658139e-246im, c = -4.495267265324774e-79 - 1.382478711138964e-74im)
-    @test orient(a,b,c) == CPredicates.orient(a, b, c)
-    @test orient(a,b,c) == ExactPredicates.orient(a,b,c)
+    (a, b, c) = (a=0.0 + 0.0im, b=0.0 + 5.495397811658139e-246im, c=-4.495267265324774e-79 - 1.382478711138964e-74im)
+    @test orient(a, b, c) == CPredicates.orient(a, b, c)
+    @test orient(a, b, c) == ExactPredicates.orient(a, b, c)
 end
 
 const floatgen = Data.Floats{Float64}()
+const floatgen2 = Data.Floats{Float64}(infs=false, nans=false) # IntervalArithmetic.jl screams out warnings with bad intervals during testing
 const complexgen = @composed function _complex(a=floatgen, b=floatgen)
-    a+b*im
+    a + b * im
+end
+const complexgen2 = @composed function _complex(a=floatgen2, b=floatgen2)
+    a + b * im
 end
-
-# @check function orient_c(a=complexgen, b=complexgen, c=complexgen)
-#     dir = CPredicates.orient(a,b,c)
-#     event!("Adaptive", dir)
-#     edir = CPredicates.orient_exact(a,b,c)
-#     event!("Exact", edir)
-#     dir == edir
-# end
 
 @check function output(a=complexgen, b=complexgen, c=complexgen)
-    orient(a,b,c) ∈ (-1, 0, 1)
+    orient(a, b, c) ∈ (-1, 0, 1)
 end
 
 # Note: This should be true for all, if not this is a bug in the port
 @check function orient_against_c(a=complexgen, b=complexgen, c=complexgen)
-    dir = orient(a,b,c)
+    dir = orient(a, b, c)
     event!("Port", dir)
-    cdir = CPredicates.orient(a,b,c)
+    cdir = CPredicates.orient(a, b, c)
     event!("Original", cdir)
     dir == cdir
 end
 
 # The next two should agree
-@check function orient_against_exact(a=complexgen, b=complexgen, c=complexgen)
-    dir = orient(a,b,c)
+@check function orient_against_exact(a=complexgen2, b=complexgen2, c=complexgen2)
+    assume!(all(x -> iszero(x) || exponent(abs(x)) ∈ -141:201, (a.re, a.im, b.re, b.im, c.re, c.im)))
```

Comment:
Can you add a comment, with a reference to the paper? Maybe also a short statement in the Readme?

Reply:
Done